### DISCUSSION TITLE
Homematic Wetterstation Pro

### DISCUSSION THREAD

**Discussion by KarlKaracho** [OpenCCU/OpenCCU#1776] [🇩🇪 Generelle Diskussionen]
Auch wenn ich Gefahr laufe, wieder nicht richtig gesucht zu haben.

Habe ich eine Frage zur Raspberrymatic und / oder zur Homematic IP Wetterstation Pro / zu Home Assistant.

Ich benutze Home Assistant zusammen mit dem HM-MOD-RPI-PCB Funkmodul. Das klappt soweit auch wunderbar.

Jetzt habe ich mir eine Homematic IP Wetterstation Pro zugelegt. Diese kann ich auch wunderbar in der Raspberrymatic auslesen, nur werden diese "Sensorwerte" nicht an Homeassistant übergeben.

Stehe ich da einfach auf dem Schlauch? Bzw. was muss ich tun / oder was mache ich falsch um diese Sensoren in Home Assistant angezeigt zu bekommen?

Ich habe sonst HMIP Heizkörper Thermostate, bei denen klappt die Übergabe einwandfrei.

**Comment by Baxxy13** [comment]
Meinst du den "Homematic IP Smart Home Funk-Wettersensor pro HmIP-SWO-PR" ?

Welche Integration wird benutzt? 
A: die Alte... [Homeassistant Homematic](https://www.home-assistant.io/integrations/homematic/)
B: die Neue... [Wiki - HomeAssistant Integration](https://github.com/jens-maus/RaspberryMatic/wiki/HomeAssistant-Integration)

RaspberryMatic selbst hat keinen Einfluss darauf wie die Geräte in Homeasssistant integriert / dargestellt werden.
Das ist Sache der Integration. 

Wenn du also A benutzt, dann wäre die beste Anlaufstelle wohl die [Home Assistant Community](https://community.home-assistant.io/).

Wenn du B benutzt (m.E. die bessere Integration) dann bist du im [Homeassistant - Bereich des Homematic-Forums](https://homematic-forum.de/forum/viewforum.php?f=84) oder [direkt bei der Integration](https://github.com/danielperna84/hahomematic/discussions) besser aufgehoben.

**Reply by KarlKaracho** [reply]
Hi,
danke für die Antwort.
Dann werde ich mich dort mal auf die Suche begeben.
